web/locales/de.json

```diff
@@ -1138,4 +1166,5 @@
             "zoneName": "Simbabwe"
           }
     }
+  }
```

Comment:
This shouldn't have been here, I missed it during cleanup, sorry. Can you remove this line?